### PR TITLE
Update README and LICENSE files

### PR DESCRIPTION
**Pull request by aggarw13**
Hygiene fixes to `README` and `LICENSE` files

By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

### CODE REVIEW

**Review by leegeth — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
 
-### v4_beta_deprecated branch
+### v4_beta_deprecated branch (Formerly, named v4_beta)
 The [v4_beta_deprecated](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v4_beta_deprecated) branch contains a beta version of the AWS IoT Embedded C SDK libraries, which is now deprecated. This branch was earlier named as *v4_beta*, and was renamed to *v4_beta_deprecated*. The libraries in this branch will not be released. However, critical bugs will be fixed and tested. No new features will be added to this branch.
 
 
 ## Releases
 
-All of the released versions of the AWS IoT Embedded C SDK libraries are available as git tags. For example, the latest v3 SDK version is available at [tag 3.1.0](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.0).
+All of the released versions of the AWS IoT Embedded C SDK libraries are available as git tags. For example, the latest v3 SDK version is available at [tag 3.0.1](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.0.1).
```
Comment (abhidixi11) [inline README.md:17]: We should not call3.01 latest, just say the last. That was released 2 years back.

**Review by leegeth — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (aggarw13) [reply README.md]: Good point, made the change

File: README.md [hunk 2]
```diff
@@ -8,13 +8,13 @@
 ### master branch
 The [master](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/master) branch hosts the continuous development of the AWS IoT Embedded C SDK libraries. Please be aware that the libraries in master branch may have bugs and performance issues. Consider using the released versions of the AWS IoT Embedded C SDK for production ready software.
 
-### v4_beta_deprecated branch
+### v4_beta_deprecated branch (Formerly, named v4_beta)
```
Comment (sarenameas) [inline README.md:11]: A comma is not needed here

**Review by abhidixi11 — APPROVED** [review]

**Review by abhidixi11 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (aggarw13) [reply README.md:11]: Wouldn't the non-comma version be `named v4_beta formerly`?

Comment (gkwicker) [reply README.md:11]: No, _"Formerly named v4_beta"_ is fine.

**Review by gkwicker — CHANGES_REQUESTED** [review]
Good to go after addressing @sarenameas's comment.

**Review by sarenameas — APPROVED** [review]

**Review by abhidixi11 — APPROVED** [review]

**Review by leegeth — APPROVED** [review]

**Pull request merged**